acl/src/main/java/org/apache/rocketmq/acl/AccessValidator.java

```diff
@@ -60,6 +63,38 @@ public interface AccessValidator {
      */
     boolean updateAccessConfig(PlainAccessConfig plainAccessConfig);
 
+    /**
+     * Update the accessKey's secretKey whiteRemoteAddress admin defaultTopicPerm and defaultGroupPerm
+     * @param plainAccessConfig
+     * @return
+     */
+    boolean updateAclAccount(PlainAccessConfig plainAccessConfig);
+
+    /**
+     * update the accessKey's resourcePerms
+      * @param accesskey
+     * @param resourceAndPerm
+     * @param operationType
+     * @return
+     */
+    boolean updateAclResourcePerms(String accesskey, ResourceAndPerm resourceAndPerm, OperationType operationType);
+
+    /**
+     * update the accessKey's namespacePerms
+     * @param accesskey
+     * @param namespaceAndPerms
+     * @param operationType
+     * @return
+     */
+    boolean updateAclNamespacePerms(String accesskey, List<NamespaceAndPerm> namespaceAndPerms, OperationType operationType);
+
+    /**
+     * get the accessKey's config
+     * @param accesskey
+     * @return
+     */
```

Comment:
IMO `PlainAccessConfig` is an implementation of ACL, it should be separated from interface `AccessValidator`.